src/sentry/models/organizationmember.py

```diff
@@ -342,7 +343,14 @@ def get_teams(self):
         )
 
     def get_scopes(self):
-        return roles.get(self.role).scopes
+        scopes = roles.get(self.role).scopes
+
+        if self.role == "member" and not self.organization.get_option(
+            "sentry:events_member_admin", EVENTS_MEMBER_ADMIN_DEFAULT
+        ):
+            scopes = frozenset(s for s in scopes if s != "event:admin")
```

Comment:
This is fine as-is but seems like it might break in the future if we added additional event scopes. Eg, if we added an "event:superadmin" scope it'd automatically get included here, giving unanticipated extra permissions to the user.

Reply:
I dont disagree, but that could be said about any change ins copes - they'd have to be updated in quite a few locations. I don't know of any other simplistic way we'd go about making this change, and imagine we'd overhaul a lot of this once we implement more fine-grained control over scopes.